cmake/pbft.cmake

```diff
@@ -65,6 +65,7 @@ if("sgx" IN_LIST TARGET)
     ${PARSED_ARGS_INCLUDE_DIRS}
     ${EVERCRYPT_INC}
   )
+  use_client_mbedtls(libbyz.enclave)
```

Comment:
This should be `ue_oe_mbedtls()`.

Reply:
In general, use_client_mbedtls() on a target whose name ends in "enclave" is unlikely to be what we want :)

Reply:
true dat